### ISSUE TITLE
The Edit button link is incorrect on translated pages.

### ISSUE THREAD

**Issue by tequdev** [XRPLF/xrpl-dev-portal#2516]
https://xrpl.org/ja/docs/references/protocol/ledger-data/common-fields/

```
https://github.com/XRPLF/xrpl-dev-portal/tree/master/ja/docs/references/protocol/ledger-data/common-fields.md
```

should be

```
https://github.com/XRPLF/xrpl-dev-portal/tree/master/%40i18n/ja/docs/references/protocol/ledger-data/common-fields.md
```

**Comment by mDuo13** [comment]
I notice there's a second problem with that page where it's not in the sidebar, either. Opened #2518 to fix the sidebar issue. For the main issue, I'm reporting the problem to Redocly.

**Comment by mDuo13** [comment]
Redocly has confirmed that it's a bug in their end and they have been able to replicate with a test case, so we can hopefully expect a fix in one of their upcoming releases. I'll leave this ticket open until we update to a version of Redocly/Realm with a fix.

**Comment by mDuo13** [comment]
Supposedly fixed in Realm v0.89.4

**Comment by tequdev** [comment]
It seems to have been fixed.
